content/v2.0/cloud/get-started.md

```diff
@@ -25,22 +25,17 @@ Once you're ready to grow, [upgrade to the Usage-based Plan](/v2.0/cloud/account
 
 ## Sign up
 
-1.  Go to [InfluxDB Cloud 2.0]({{< cloud-link >}}), enter your email address and password,
-    and click **Sign Up**.
-2.  InfluxDB Cloud requires email verification to complete the sign up process.
-    Verify your email address by opening the email sent to the address you provided
-    and clicking **Verify Your Email**.
-3.  Select a region for you {{< cloud-name >}} instance.
-    Currently, {{< cloud-name >}} AWS - US West (Oregon) is the only region available.
+1. Go to [InfluxDB Cloud 2.0]({{< cloud-link >}}), enter your email address and password, and click **Sign Up**.
+2. InfluxDB Cloud requires email verification to complete the sign up process. Verify your email address by opening
+   the email sent to the address you provided and clicking **Verify Your Email**.
+3. Choose your cloud provider. Currently, **AWS** is the only cloud provider available.
+4. Select a region for your {{< cloud-name >}} instance. Currently, **US West (Oregon)** and **EU Frankfurt** are available.
     _To suggest regions to add, click **Let us know** under Regions._
```

Comment:
Just for maintainability reasons, should we turn this into a nested list?

~~~
4. Select a region for your {{< cloud-name >}} instance. The following regions are available:

    - **US West (Oregon)**
    - **EU Frankfurt**

    _To suggest regions to add, click **Let us know** under Regions._
~~~